src/main/java/pages/AddClientProceedsPage.java

```diff
@@ -0,0 +1,239 @@
+    private WebElement countryDroplistButton;
+
+    @FindBy(xpath = "(//span[contains(@role,'combobox')])[13]")
+    private WebElement countyDroplistButton;
+
+    @FindBy(xpath = "//input[@id='input-city']")
+    private WebElement cityField;
+
+    @FindBy(xpath = "//input[@id='input-address']")
+    private WebElement adressField;
+
+    @FindBy(xpath = "(//span[@role='combobox'])[14]")
+    private WebElement currencyButton;
+
+    @FindBy(xpath = "//input[@id='input-contactName']")
+    private WebElement contactPersonField;
+
+    @FindBy(xpath = "//input[@id='input-email']")
+    private WebElement contactPersonEmailField;
+
+    @FindBy(xpath = "//input[@id='input-phone']")
+    private WebElement contactPersonPhoneField;
+
+    @FindBy(xpath = "//input[@id='input-website']")
+    private WebElement contactPersonWebsiteField;
+
+    @FindBy(xpath = "//button[contains(@class,'btn btn-sm btn-primary')][contains(text(),'Salvează')]")
+    private WebElement saveButton;
+
+    @FindBy(xpath = "(//button[contains(@type,'button')][normalize-space()='Închide'])[2]")
+    private WebElement cancelButton;
+
+    @FindBy(xpath = "//div[@class='otFlat bottom ot-wo-title vertical-align-content']")
+    private WebElement cookiesAdd;
+
+    @FindBy(xpath = "//button[@class='onetrust-close-btn-handler onetrust-close-btn-ui banner-close-button ot-close-icon']")
+    private WebElement closeCoockiesAddButton;
+
+    @FindBy(xpath = "//div[@class='toast toast-success']")
+    private WebElement confirmationPopup;
+
+    @FindBy(xpath = "//div[@class='card   ']")
+    private WebElement paymentCard;
+
+    @FindBy(xpath = "//div[@class='col-12 text-center']//a")
+    private WebElement paymentButton;
+
+    private WebDriverWait waitPage(){
+        return  new WebDriverWait(driver,15);
+    }
+
+    public void closeCoockiesAdd(){
+        find(cookiesAdd);
+        click(closeCoockiesAddButton);
+    }
+
+    public void clickContAppPSButton(){
+        click(contAppPSButton);
+    }
+
+    public void succesConfirmationPopup(){
+        waitPage().until(ExpectedConditions.visibilityOf(confirmationPopup));
+    }
+
+    public void clickAddProceedsButton(){
+        click(addProceedsButton);
+    }
+    public void clickClientDroplist(){
+        click(clientDroplistButton);
+    }
+    public WebElement clientDropDownList(int index, String selector){
+        List<WebElement> clients = driver_local.findElements(By.cssSelector(selector));
+        WebElement firstClient = clients.get(index);
+        return firstClient;
+    }
+
+    public WebElement byWordDropList(String selector, String text){
+        waitPage();
+        List<WebElement> fields = driver_local.findElements(By.cssSelector(selector));
+        WebElement fieldName = driver_local.findElement(By.xpath("//*[text()='" + text + "']"));
+        return fieldName;
+
+    }
+
+    public WebElement byWOrdDropList(String selector, String text){
```

Comment:
ai deja metoda asta in BaseTest. Te rog sa corectezi

Reply:
public void selectCurrencyDroplist(String text) {
        click(currency);
        click(listofElements(currencyDroplist, text));
    }